Add fake data file

eng-somaia opened the pull request.

eng-somaia committed:
Doc: add fake data file
add a js file tat contains 5 doctors info and 5 articles.

eng-somaia committed:
Fix: add a doctorImg property to psychiatrists array.

MuhannedNoman commented:
@eng-somaia Looks like the test has failed. Can you check that out please, and then tag me again once done.

eng-somaia committed:
Fix: add a missing , in psychiatrists array.

eng-somaia commented:
> @eng-somaia Looks like the test has failed. Can you check that out please, and then tag me again once done.

@MuhannedNoman  Passing tests.

MuhannedNoman commented on fakeData.js:
```diff
@@ -0,0 +1,93 @@
+const psychiarists = [
+    {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
```
Let's edit this to image only.

MuhannedNoman commented on fakeData.js:
```diff
@@ -0,0 +1,93 @@
+    {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
```
From the design looks like we will only display one thing. So Let's reflect that her and just pick one thing to add.

MuhannedNoman commented on fakeData.js:
```diff
@@ -0,0 +1,93 @@
+    }
+];
+const articles = [
+    {
```
We need to add date here since it is displayed in the Article page and in home page.

MuhannedNoman commented on fakeData.js:
```diff
@@ -0,0 +1,93 @@
+const psychiarists = [
```
Let's add an ID for each element here.

MuhannedNoman replied:
Article included.

MuhannedNoman commented:
Impressive work here @eng-somaia just left you some comments. Let me know what you think !

MuhannedNoman commented:
One last thing let's add a folder in the `src` folder and call it `fake[object]Services.js` ex `fakeDoctorsServices.js`
Inside the file we will only have Doctor's  data. We will use ES6 Export keyword without the word module. Also, add a function and export it call it `get[Object]` that will return the whole data. 
This way we can add and edit stuff in this single file in the future. 
Ex
```
export const doctors = [{},{},{}]

export function getDocotrs() {
     return doctors.filter(d => d)
}

```

eng-somaia committed:
Fix: edit fake data file and add fake doctors services
add id and date properties to articles array, add id property to doctors array, change doctorImg to image in doctors array, and move doctors array to fake doctors services file.

eng-somaia commented:
@MuhannedNoman Edit all the things you mentioned above. Let me know if there is anything to add or edit.

MuhannedNoman reviewed (changes requested):
LGTM, one thing though. 
Change `fakeData.js` to `fakeArticleService.js` and move the two files into a folder called `services` inside the `src` folder. Once that is implemented feel free to merge.

eng-somaia committed:
Fix: add services folder
rename fakeDate flie to fakeArticleService file and move it along with the fakeDoctorsService file to services folder in src folder.

eng-somaia commented:
Done.
